basex/src/test/java/com/formulasearchengine/mathosphere/basex/TexQueryGeneratorTest.java

```diff
@@ -115,7 +115,7 @@ public void testErrorHandling () throws Exception {
 		try {
 			t.request( "E=mc^2" );
 		} catch( final IOException expected ) {
-			assertEquals( "com.fasterxml.jackson.core.JsonParseException",
+			assertEquals( "java.io.IOException",
```

Comment:
@Test has an expected exception attribute or you can even use @Expected in junit4/5.

Reply:
junit 5 is very nice. I disliked juint  <5